tools/wptrunner/wptrunner/metadata.py

```diff
@@ -57,8 +56,8 @@ def get_properties(properties_file=None, extra_properties=None, config=None, pro
 
     :param properties_file: Path to a JSON file containing properties.
     :param extra_properties: List of extra properties to use
-    :param config: (deprecated) wptrunner config
-    :param Product: (deprecated) product name (requires a config argument to be used)
+    :param config: (deprecated, unused) wptrunner config
```

Comment:
Can we get away with just dropping the argument? Is this considered public wptrunner API?

Reply:
Yes, this is public API. In practice I don't know if anyone is really depending on it, but in theory an API change here would require an RFC.

For now I've added a warning.